frontend/src/metabase/query_builder/components/notebook/steps/LimitStep/LimitStep.tsx

```diff
@@ -16,15 +17,19 @@ function LimitStep({
   const { stageIndex } = step;
 
   const limit = Lib.currentLimit(query, stageIndex);
-  const value = typeof limit === "number" ? limit : "";
+  const [value, setValue] = useState(typeof limit === "number" ? limit : "");
 
-  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+  const handleBlur = (e: React.FocusEvent<HTMLInputElement>) => {
```

Comment:
1. Please destructure React imports :pray: 
2. Can we use meaningful names? :pray: `e` -> `event`

_This comment applies to `handleChange` as well._


```suggestion
  const handleBlur = (event: FocusEvent<HTMLInputElement>) => {
```

Reply:
Yes.
I'll keep this in mind going forward because you already gave this advice.